tools/ports/binaryen.py

```diff
@@ -8,14 +8,12 @@
 TAG = 'version_59'
 
 def needed(settings, shared, ports):
-  if not settings.WASM: return False
-  try:
-    if shared.BINARYEN_ROOT: # if defined, and not falsey, we don't need the port
-      logging.debug('binaryen root already set to ' + shared.BINARYEN_ROOT)
-      settings.BINARYEN_ROOT = shared.BINARYEN_ROOT
-      return False
-  except:
```

Comment:
This try/catch was here because  shared.BINARYEN_ROOT used to not exist at all in some cases.  Nowadays is guaranteed to exist, but could be None